Package.swift

```diff
@@ -9,23 +9,26 @@ let package = Package(
     products: [
         .library(
             name: "ZXingCppWrapper",
-            type: .static,
+            type: .dynamic,
             targets: ["ZXingCppWrapper"])
     ],
     targets: [
-        .binaryTarget(
+        .target(
             name: "ZXingCpp",
-            path: "ZXingCpp.xcframework"
+            path: "core/src",
+            publicHeadersPath: "."
         ),
         .target(
             name: "ZXingCppWrapper",
             dependencies: ["ZXingCpp"],
-            path: "Sources/Wrapper",
+            path: "wrappers/ios/Sources/Wrapper",
             publicHeadersPath: ".",
-            cxxSettings: [
-                .unsafeFlags(["-stdlib=libc++"]),
```

Comment:
Short remark: not necessary anymore due to dynamic build. I didn't need it for the static variant either though.